Data/Sys/GameSettings/R92.ini

```diff
@@ -0,0 +1,14 @@
+# R92E01, R92P01, R92J01 - Pikmin 2 Wii
+[Core]
+# Values set here will override the main Dolphin settings.
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId = 4
+EmulationIssues = Distorted sound effects on DSP HLE. Use DSP LLE to fix.
```

Comment:
I took care of the whitespace issue. I did not test the audio myself. I simply added that note to be consistent with the [wiki page](https://wiki.dolphin-emu.org/index.php?title=Pikmin_2_%28Wii%29). If it should be removed, I'd be happy to do so.